.woke.yaml

```diff
@@ -0,0 +1,7 @@
+# Ignore ingress charm library as it uses this non compliant terminology: whitelist.
+ignore_files:
+ - lib/charms/nginx_ingress_integrator/v0/ingress.py
```

Comment:
Can I suggest the same changes here as in https://github.com/canonical/discourse-k8s-operator/pull/10 ?